spec/06-expressions.md

```diff
@@ -1746,27 +1739,18 @@ a sub-expression of parameterless method type, is not evaluated in the expanded
 
 ### Dynamic Member Selection
 
-The standard Scala library defines a trait `scala.Dynamic` which defines a member
-`applyDynamic` as follows:
+The standard Scala library defines a marker trait `scala.Dynamic`. Subclasses of this trait are able to intercept selections and applications on their instances by defining methods of the names `applyDynamic`, `applyDynamicNamed`, `selectDynamic`, and `updateDynamic`. Under the conditions of [implicit conversion](#dynamic-member-selection), the following rewrites are performed, assuming $e$ has type `Dynamic`, and the originally expression does not type check under normal rules:
```

Comment:
Can you maybe clarify / expand a little what "Under the conditions of implicit conversion" means?